app/src/main/index.js

```diff
@@ -545,125 +543,133 @@ async function main() {
   // handle ipc messages from the renderer process
   handleIPC()
 
-  if (!MOCK) {
```

Comment:
Only no connecting to a node now. Keeping the init flow so connecting later is not a hassle.